actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java

```diff
@@ -963,6 +987,25 @@ private void sendEmailAndSms(Map<String, Object> userMap) {
     tellToAnother(EmailAndSmsRequest);
   }
 
+  private Response sendResetPasswordLink(Map<String, Object> userMap) {
```

Comment:
Isn't this already somewhere? If so, please remove that and unify these code.

Reply:
I searched no-where in our code base this is present. From ResetPasswordController "resetPassword" actor is called. In this use-case we need reset-password link to be generated to called from here.